client/client.go

```diff
@@ -295,6 +339,9 @@ func newHttpsClient(conf Config) (*http.Client, error) {
 	transport := http.Transport{
 		TLSClientConfig: &tlsc,
 		Proxy:           http.ProxyFromEnvironment,
+		DialTLS: func(network string, addr string) (net.Conn, error) {
```

Comment:
The docs recommend using `DialTLSContext`: 
```
// DialTLS [...]
// Deprecated: Use DialTLSContext instead, which allows the transport
// to cancel dials as soon as they are no longer needed.
// If both are set, DialTLSContext takes priority.
```
However, I don't think we use request contexts within the client so I'm not sure if this is relevant.

Reply:
no. Alf, we have to support older go, thats why I use the deprecated method, and also that is why I added the comment about ts.EnableHTTP2 above.

Reply:
On a second thought I agree as well. Besides, I just discovered that the openssl library has hardcoded `net.Dial` anyway...